libbeat/processors/add_cloud_metadata/provider_aws_ec2.go

```diff
@@ -22,13 +22,14 @@ import (
 	"fmt"
 	"net/http"
 
+	"github.com/elastic/elastic-agent-libs/logp"
```

Comment:
nit: I dont think we need to move this import around

Reply:
We need to move it around because of the errors of one of the tests. They force it to be sorted through the tool `goimport`